diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back end manages, saves and stores information in a database for the app which can be accessed by users at any given time.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We have serializers that replace all the views needed for our apps.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Model
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+POST, GET, PATCH, PUT, DELETE
```

Comment:
These are the HTTP verbs we wanted the rails actions that they map to:

`Index, Show, Create, Update, Destroy`